### ISSUE TITLE
install error (can't find plugins directory?)

### ISSUE THREAD

**Issue by jots** [weepy/kaffeine#28]
after a fresh install of sudo npm  install kaffeine:
(i also tried with -g)

~/node_modules/kaffeine/bin> ./kaffeine

~/node_modules/kaffeine/bin/command.js:13
    for(var i =0; i< files.length;i++) {
                          ^
TypeError: Cannot read property 'length' of undefined
    at ~/node_modules/kaffeine/bin/command.js:13:27

~/node_modules/kaffeine/bin> node --version
v0.5.0-pre

~/node_modules/kaffeine/bin> npm --version
1.0.5

it looks like it's having problems finding the plugins directory.

**Comment by weepy** [comment]
yes. I broke the command line a while ago. will fix this coming weekend.
